rules/manager.go

```diff
@@ -49,7 +45,7 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type", "status"},
```

Comment:
The original line of argument was that it doesn't matter because (an a healthy system/set-up) errors will only ever be a tiny fraction of the total number of requests. So you could discuss forever if you want to include them or not, but quantile latency will not change anyway. _Unless_ your fraction of errors is high. Then you either have a real problem anyway, so exact quantile latency doesn't really matter, or you have a set-up where a certain high error rate is expected, in which case you might want to treat them separately because latency then actually matters.